src/store/consts/static.ts

```diff
@@ -367,3 +367,5 @@ export const SIGNING_SNACKBAR_CONFIG: Omit<ISnackbar, 'open'> = {
   variant: 'pending',
   persist: true
 }
+
+export const STABLECOIN_SYMBOLS: string[] = ['USDC', 'USDT']
```

Comment:
It will be better to rely on addresses rather than tickers. We want to do this only in specific cases